### ISSUE TITLE
WARNING:root:NaN or Inf found in input tensor.

### ISSUE THREAD

**Issue by 1402686212** [Kebii/Zoom-Transformer#2]
Dear author, thank you very much for your contribution. I have a question: after the code and data you provided are run, the warning shown in the figure is displayed. The loss is always "nan". This is obviously a case of gradient explosion. I tried to change the learning rate to be smaller. A little bit, but it still doesn't work. What's the reason?
![image](https://github.com/Kebii/Zoom-Transformer/assets/28975368/1bad56a5-6596-4d7f-a878-3d15c198a5e0)

**Comment by TrinityNeo99** [comment]
I encountered the same problem, and I set  `normalization: False` on the train,yaml. It seems everything work right and this model is well suitable for my resarch
